.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,49 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - main
+  pull_request:
+
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - uses: actions/checkout@v3
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@07db5389c99593f11ad7b44463c2d4233066a9b1
+        with:
+          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
```

Comment:
I think we should remove all `Optional` keys, since we would likely lookup the docs if we ever want to reconfigure.

Reply:
Indeed. We added them to testcontainers-go in https://github.com/testcontainers/testcontainers-go/issues/569, but we should remove it to clean up the file